.travis.yml

```diff
@@ -43,11 +43,12 @@ script:
     - cat openerp/addons/base/migrations/*/tests/*.yml addons/*/migrations/*/tests/*.yml > ../test_data80.yml
     # Roundtrip to previous release to update the test database with additional test data
     # Loading and committing test data without triggering post tests needs https://github.com/odoo/odoo/pull/13146. This is now included in OpenUpgrade.
+    - old_commit="$(git rev-parse HEAD)"
```

Comment:
@StefanRijnhart @mreficent FYI a possible workaround for avoiding the problem we have with the Travis glitch

Reply:
good to know

Reply:
should we implement it in the other versions?

Reply:
We are experimenting in this branch, but if everything goes alright, yes, we will propagate it to all branches.

Reply:
Cherry-picked in all branches after successful tests in 9.0 branch.

Reply:
Happy to help :blush: